app/mailers/rock_pebble_mailer.rb

```diff
@@ -0,0 +1,56 @@
+class RockPebbleMailer < ApplicationMailer
+  def confirmation_message(info)
+    address = info[:rock_email]
+    name = info[:pebble_name]
+    @message = "#{name} has requested you as a Rock through the Rock and Pebble
+                Program on Paired.tech. Please visit #{paired_link} to accept or deny
```

Comment:
On this line, I would suggest changing "Program" to "feature," to make it sound a little less like something we're actively managing; and "deny" to "decline" to match the terminology we're using elsewhere.